index.html

```diff
@@ -7,24 +7,45 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Awesome Books</title>
   <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
     integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
 </head>
 
 <body>
-  <h1 class="catch">Awesome Books</h1>
+  <div>
+  </div>
   <div class="container">
-    <h1 class="display"></h1>
-    <div class="books">
+    <div class="display">
+      <h1>All Awesome Books</h1>

```

Comment:
- [ ] **In implementing the changes, the alignment of your "All Awesome Books" and "Add Book" headings is mistakenly disturbed. Please fix it to match the wireframe**

Cloned Design|Original Design
:------------------:|:-------------------:
![image](https://user-images.githubusercontent.com/59168430/124805446-246b9800-df79-11eb-8522-403e85938797.png)|![image](https://user-images.githubusercontent.com/59168430/124805539-482ede00-df79-11eb-81b5-854f22a8f9b7.png)